data/wacom.metainfo.xml.in

```diff
@@ -31,6 +31,14 @@
   <update_contact>contact_at_elementary.io</update_contact>
 
   <releases>
+    <release version="8.0.1" date="2024-12-17" urgency="medium">
+      <description>
+        <ul>
+          <li>A fix for a crush that occured when stylus is not detected by libwacom</li>
+        </ul>
+      </description>
+    </release>
+
```

Comment:
You should put this into the `issues` tag:

```
    <release version="8.0.1" date="2024-12-17" urgency="medium">
      <description>
        <p>Updated translations</p>
      </description>
      <issues>
        <issue url="https://github.com/elementary/switchboard-plug-wacom/issues/69">Clicking on the wacom plug crushes The switchboard</issue>
      </issues>
    </release>
```

Reply:
Done.